docs/index.md

```diff
@@ -8,6 +6,7 @@ bugs at runtime.
 
 The analyzer works in Visual Studio 2015 Update 1 or later. Install one of these
 NuGet packages in each project that needs it:
+

```

Comment:
A bonus change. With this, the list renders as a list. I can revert if you like

Reply:
No, it's fine. Strange we didn't notice this sooner